datastore/src/main/scala/org/allenai/datastore/Datastore.scala

```diff
@@ -1,12 +1,14 @@
 package org.allenai.datastore
 
+import com.amazonaws.event.{ ProgressEvent, ProgressListener }
```

Comment:
Hrm. I thought IntelliJ would do this for me. I just auto-organized all the imports in the project. Maybe it'll do it once it can start from a clean state.